src/Core/maps/src/Core/IGeoPathMapElement.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using Microsoft.Maui.Devices.Sensors;
+
+namespace Microsoft.Maui.Maps
+{
+	public interface IGeoPathMapElement : IMapElement
```

Comment:
should this be

```C#
public interface IGeoPathMapElement : IMapElement, IList<Location>
```

Instead of having a `GeoPath` property?

Reply:
OK, moved, But we keep what we had on Controls right?

Reply:
yea, it's probably not worth breaking that API. As much as I'd like to get rid of that property or change it to something like `Points`.